src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -776,6 +776,11 @@ private function hasProductOptionWithNameAndValue(string $expectedOptionName, st
 
         foreach ($productVariants as $productVariant) {
             foreach ($productVariant['optionValues'] as $optionValueIri) {
+
+                if (is_array($optionValueIri)) {
+                    continue;
+                }
```

Comment:
Why?

Reply:
It was some test fix for `viewing_products` scenario. Reverted it since `view_products` scenario is not enabled after all